src/gui/MainWindow.cpp

```diff
@@ -648,6 +672,11 @@ void MainWindow::captureEmpty()
 {
 	setEnablements(false);
 	resizeToAnnotator();
+	if (!mToolBar->isVisible()) {
+		// Show temporarily; will hide again when capture changes
+		mToolBar->setVisible(true);
```

Comment:
If the docks are hidden when the last tab is closed, MainWindow will disappear. That's why, as an workaround, I have added the call to setVisible(true). 
Later, when the capture is changed, docks will be hidden again.